app/views/LicensorViewComponent.vue

```diff
@@ -222,6 +222,7 @@ module.exports = Vue.extend({
             user = yield api.users.getByEmail({email})
             attrs = data
             attrs.maxRedeemers = parseInt(data.maxRedeemers)
+            data.endDate = data.endDate + " " + "23:59"   # Otherwise, it ends at 12 am by default which does not include the date indicated
```

Comment:
Can we use the `attrs` variable instead of the `data` variable here and the next 2 lines, like AdministerUserModel.coffee?
`attrs.endDate = attrs.endDate + " " + "23:59"`

Reply:
Yeah, we can do that to make it similar to AdministerUserModel.coffee.
Do you want me to change it in this PR?

Reply:
Yea, if you're going to change it please do it in the PR so you can squash and merge into one commit in master and keep things clean.  Thanks!